docs/integrations/bitbucket/locations.md

```diff
@@ -19,34 +19,4 @@ Please see [the Bitbucket Cloud documentation](../bitbucketCloud/locations.md).
 
 ## Bitbucket Server
 
-```yaml
```

Comment:
at this point, we could delete this page entirely I think

Reply:
PS: maybe we should mark the processor as deprecated, too. This was not possible so far as only Bitbucket Cloud was covered. But with this change, we will cover both cases.

Reply:
I removed the page, however I'm not sure if that's correct.
Even if it's deprecated, shouldn't we keep the documentation but mark it as deprecated and point to the new integration & entity processors?